index.js

```diff
@@ -5,16 +5,8 @@ class SortedList {
   }
 
   add(item) {
-    if(item > this.items[this.length - 1] || this.length === 0) {
-      this.items.push(item);
-    }
-
-    for (let i = 0; i < this.length; i++) {
-      if(this.items[i] > item) {
-        this.items.splice(i, 0, item);
-        break;
-      }
-    }
+    this.items.push(item);
```

Comment:
add(item) {
    if (item !== undefined) {
      // use .push() to add element to the array
      this.items.push(item);
      // add +1 on the length of the array
      this.length++;
      this.items.sort((a, b) => a - b);
    }
    return this.items;
  }